packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -47,7 +47,7 @@ export const getBootstrapUrl = (region: string) =>
  */
 export class FileWatchingSandbox extends EventEmitter implements Sandbox {
   private watcherSubscription: Awaited<ReturnType<typeof subscribe>>;
-  private outputFilesExcludedFromWatch = ['.amplify'];
+  private outputFilesExcludedFromWatch = ['.amplify', '**/amplify/*_env.ts'];
```

Comment:
I can try putting something together for this

Reply:
Another thing is the entry to `.gitignore` has the coupling and would have to be updated if we ever make changes. This one is more ok because developers will notice if the type def files stop being ignored.